tests/spec/ScraperSpec.js

```diff
@@ -27,6 +29,48 @@ describe('Scraper', function () {
   afterEach(function () {
     while (document.body.firstChild) document.body.removeChild(document.body.firstChild)
   })
+  afterEach(function () {
+    sandbox.restore()
+  })
+
+  it('Should handle error on job', function (done) {
+    sandbox.stub(Job.prototype, 'execute')
+      .callsFake(function (params, callback) {
+        callback(new Error('Fake error test'))
+      })
+    var b = document.querySelector('#scraper-test-one-page a')
```

Comment:
We're using ES6 `let` and `const` in this file so let's keep using them here, too